lib/model_patches.rb

```diff
@@ -54,7 +54,9 @@ def email_subject_request(opts = {})
 
         def late_calculator
           @late_calculator ||=
-            if public_body.has_tag?('school')
+            if public_body.has_tag?('scotland')
+              orig_late_calculator
```

Comment:
Shouldn't this be:
```suggestion
              ScotlandLateCalculator.new
```

Reply:
Oh oops! added that to test it out and clearly to eager on the `git add .` 😊